docs/genai/howto/install.md

```diff
@@ -23,61 +29,149 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
 
 Ensure that the `CUDA_PATH` environment variable is set to the location of your CUDA installation.
 
+
 ## Python packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these sets of packages (CPU, DirectML, CUDA) should be installed in your environment.
 
 ### CPU
 
+#### x64
+
 ```bash
-pip install numpy
+# Install ONNX Runtime nightly. Update this to the released version when it is available.
+pip install ort-nightly --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
+
+pip install onnxruntime-genai --pre
+```
+
+#### Arm64
+
+```bash
+# Install ONNX Runtime nightly QNN version (which supports Arm CPU). Update this to the released version when it is available.
+pip install ort-nightly-qnn --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
 pip install onnxruntime-genai --pre
 ```
 
-### DirectML
 
-Append `-directml` for the library that is optimized for DirectML on Windows
+### DirectML
 
 ```bash
-pip install numpy
+# Install the nightly version of ONNX Runtime. Update this to the released version when it is available.
+pip install ort-nightly-directml --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
 pip install onnxruntime-genai-directml --pre
 ```
 
 ### CUDA
 
-Append `-cuda` for the library that is optimized for CUDA environments
-
 #### CUDA 11
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
+# Install the nightly version of ONNX Runtime. Update this to the released version when it is available.
+pip install ort-nightly-gpu --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-cuda-11-nightly/pypi/simple
+
+
+# Install onnxruntime-genai built for CUDA 11
+pip install onnxruntime-genai-cuda-11 --pre
```

Comment:
We do not have any cuda 11 package published for rcs. It will only be cuda 12.
Do we intend to also publish cuda 11 packages?

Reply:
Updated